src/com/akshay/captain/jira.main.kts

```diff
@@ -39,21 +39,28 @@ private val bugFixes = "Bug fixes"
 private val instrumentation = "Instrumentation"
 private val labels = "Labels"
 
+//env vars
+private val userToken = "userToken"
+private val userName = "userName"
+
 //Jira Variables
 private val clueJiraUrl = "https://biowink.atlassian.net"
 private val readyForReleaseFilter = "filter=15768"
 
 main()
 
 fun main() {
-    val platform = args.firstOrNull()?.toPlatform() ?: Platform.Android
 
-     createReleaseVersion()
-    val cred = BasicCredentials("","")
+    val platform = args.firstOrNull()?.toPlatform() ?: Platform.Android
+    val cred = BasicCredentials(System.getenv()[userName], System.getenv()[userToken])
```

Comment:
are we missing steps in the readme to add the usertoken?

Reply:
Yes usertoken thing will add at last! hence this was not in doc